includes/models/model.llms.course.php

```diff
@@ -236,9 +237,14 @@ public function get_difficulty( $field = 'name' ) {
 
 			foreach ( $terms as $term ) {
 
-				return $term->$field;
+				if ( property_exists( $term, $field ) ) {
 
+					$difficulties = wp_list_pluck( $terms, $field );
+
+				}
 			}
```

Comment:
why are we keeping the foreach?
You're doing the same `wp_list_pluck( $terms, $field );` for each `$term` in `$terms`.
I see you want to make sure the property exists, but we didn't do that check before so, as I already said, we can live not doing that check. The whole foreach then becomes just:
`$difficulties = wp_list_pluck( $terms, $field );`

Reply:
thanks! fixed now